torch/_meta_registrations.py

```diff
@@ -34,6 +35,8 @@ def add_func(op):
                 )
                 _meta_lib_dont_use_me_use_register_meta.impl(name, f)
 
+            op.py_impl(torch._C.DispatchKey.Meta)(f)
```

Comment:
This needs to make its way upstream

Reply:
One q I have: it's a bit hard to keep straight the number of different implementations / tables of decomps that we have floating around.

Would it make sense to have `FakeTensor` start to always run with `PythonDispatcher` enabled? Then it can rely on grabbing python decomps/meta functions from the normal dispatcher machinery, and not have to use the explicit `meta_table` and `decomposition_table`.

I could maybe see a counter argument that explicit tables in python are easier to debug / reason about (just at the cost of having multiple implementations floating around in different places)

Reply:
I was definitely not a fan of fake tensor directly going to meta tensor but there was no better alternative at the time. However, torch dispatch implementations going to a table seems to be a lot more intuitive for folks than the dispatcher mechanism. Not sure what to do